go-controller/pkg/cni/cni.go

```diff
@@ -120,39 +120,21 @@ func (pr *PodRequest) cmdAdd() *PodResult {
 		logrus.Errorf("failed to parse bandwidth request: %v", err)
 		return nil
 	}
-
-	var interfacesArray []*current.Interface
-	interfacesArray, err = pr.ConfigureInterface(namespace, podName, macAddress, ipAddress, gatewayIP, config.Default.MTU, ingress, egress)
-	if err != nil {
-		logrus.Errorf("Failed to configure interface in pod: %v", err)
-		return nil
-	}
-
-	// Build the result structure to pass back to the runtime
-	addr, addrNet, err := net.ParseCIDR(ipAddress)
-	if err != nil {
-		logrus.Errorf("failed to parse IP address %q: %v", ipAddress, err)
-		return nil
-	}
-	ipVersion := "6"
-	if addr.To4() != nil {
-		ipVersion = "4"
-	}
-	result := &current.Result{
-		Interfaces: interfacesArray,
-		IPs: []*current.IPConfig{
-			{
-				Version:   ipVersion,
-				Interface: current.Int(1),
-				Address:   net.IPNet{IP: addr, Mask: addrNet.Mask},
-				Gateway:   net.ParseIP(gatewayIP),
-			},
-		},
-	}
-
+	podIntfaceInfo := &PodIntfaceInfo{
```

Comment:
"Intface" is weird... Usually people abbreviate "interface" to "iface", but you could also just not abbreviate it

Reply:
done.